docs/source/api/apollo-server.md

```diff
@@ -345,10 +345,26 @@ addMockFunctionsToSchema({
 
 *  `privateVariables`: Array<String> | boolean
 
+   DEPRECATING IN VERSION XX.XX.XX for `enforcePrivateVariables`, which will support the same
+   functionalities but allow for more flexibility.
+   
    A case-sensitive list of names of variables whose values should not be sent
    to Apollo servers, or 'true' to leave out all variables. In the former
    case, the report will indicate that each private variable was redacted in
    the latter case, no variables are sent at all.
+   
+* `enforcePrivateVariables`: ((variables: Record<string, any>, operationString?: string) => Record<string, any>) | Array<String> | boolean
+    
+    A custom function for modifying variable values, a case-sensitive list of names 
+    of variables whose values should not be sent to Apollo servers, 
+    or `true` v. `false` to blacklist v. whitelist all variables, respectively. In the first
```

Comment:
nit: wording — we try to use `safelist` or `blocklist` usually in other features
(found slack ref: https://meteor.slack.com/archives/C936DTZEG/p1527247450000377)

Reply:
Thanks for letting me know!!